configs/default/PYDR-PYRODRONEF4PDB.config

```diff
@@ -0,0 +1,90 @@
+# Betaflight / STM32F405 (S405) 4.1.1 Nov 15 2019 / 12:54:53 (1e5e3d369) MSP API: 1.42
+
+board_name PYRODRONEF4PDB
+manufacturer_id PYDR
+
+# resources
+resource BEEPER 1 B05
+resource MOTOR 1 B01
+resource MOTOR 2 B00
+resource MOTOR 3 C09
+resource MOTOR 4 A08
+resource LED_STRIP 1 B08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 5 C12
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 5 D02
+resource SERIAL_RX 6 C07
+resource INVERTER 1 C03
+resource LED 1 B04
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource ESCSERIAL 1 B09
+resource CAMERA_CONTROL 1 B09
+resource ADC_BATT 1 C02
+resource ADC_CURR 1 C01
+resource PINIO 1 A15
+resource PINIO 2 C15
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+resource BARO_CS 1 C14
+resource FLASH_CS 1 C13
+
+# timer
+timer B09 AF3           # pin B09: TIM11 CH1 (AF3)
+timer B01 AF2           # pin B01: TIM3 CH4 (AF2)
+timer B00 AF3           # pin B00: TIM8 CH2N (AF3)
+timer C09 AF3           # pin C09: TIM8 CH4 (AF3)
+timer A08 AF1           # pin A08: TIM1 CH1 (AF1)
+timer B08 AF2           # pin B08: TIM4 CH3 (AF2)
+
+# dma
+dma ADC 1 1             # ADC 1: DMA2 Stream 4 Channel 0
+dma pin B01 0           # pin B01: DMA1 Stream 2 Channel 5
+dma pin B00 0           # pin B00: DMA2 Stream 2 Channel 0
+dma pin C09 0           # pin C09: DMA2 Stream 7 Channel 7
+dma pin A08 1           # pin A08: DMA2 Stream 1 Channel 6
+dma pin B08 0           # pin B08: DMA1 Stream 7 Channel 2
+
+# feature
+feature OSD
+
+# aux
+aux 0 40 255 900 2100 0 0               // turn on vtx PIT switch 
+aux 1 41 255 900 2100 0 0               // turn on BT module
+
+
+# serial
+serial 3 1 19200 57600 0 115200        // set up for BT module
+
+
+# master
+set mag_hardware = NONE
```

Comment:
There is no reason to set this to `NONE`. It will just make it harder for anybody who might want to add an external I2C magnetometer in the future.

Speaking of which, I notice you haven't defined any I2C bus pins. Since I2C2 is an alternate function for USART3 you could add:
```
resource I2C_SCL 2 B10
resource I2C_SDA 2 B11
```
to make it easier for customers. It's a common support question about how to add a magnetometer via I2C and many board targets make it hard for the customer. Additionally you can add:
```
set mag_i2c_device = 2
```
Then all the user would need to do is connect the SCL/SDA pins to UART3 and the magnetometer will work.

Reply:
got it. making change now. thanks for the suggestion.